app/src/main/java/com/onewelcome/showcaseapp/feature/userauthentication/implicitauthentication/ImplicitAuthenticationScreen.kt

```diff
@@ -0,0 +1,259 @@
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.onFailure
+import com.github.michaelbull.result.onSuccess
+import com.onegini.mobile.sdk.android.model.entity.UserProfile
+import com.onewelcome.core.components.SdkFeatureScreen
+import com.onewelcome.core.components.ShowcaseCard
+import com.onewelcome.core.components.ShowcaseFeatureDescription
+import com.onewelcome.core.components.ShowcaseStatusCard
+import com.onewelcome.core.components.ShowcaseTooltip
+import com.onewelcome.core.theme.Dimensions
+import com.onewelcome.core.theme.toErrorResultString
+import com.onewelcome.core.util.Constants
+import com.onewelcome.showcaseapp.R
+
+@Composable
+fun ImplicitAuthenticationScreen(
+    navController: NavController,
+    viewModel: ImplicitAuthenticationViewModel = hiltViewModel()
+) {
+    ImplicitAuthenticationScreenContent(
+        uiState = viewModel.uiState,
+        onNavigateBack = { navController.popBackStack() },
+        onEvent = { viewModel.onEvent(it) }
+    )
+}
+
+@Composable
+private fun ImplicitAuthenticationScreenContent(
+    onNavigateBack: () -> Unit,
+    uiState: ImplicitAuthenticationViewModel.State,
+    onEvent: (ImplicitAuthenticationViewModel.UiEvent) -> Unit,
+) {
+    //ListenForPinNavigationEvent(navigationEvents, onNavigateToPinScreen)
+    LoadData(onEvent)
+    SdkFeatureScreen(
+        title = stringResource(R.string.implicit_authentication),
+        onNavigateBack = onNavigateBack,
+        description = {
+            ShowcaseFeatureDescription(
+                description = stringResource(R.string.implicit_authentication_description),
+                link = Constants.DOCUMENTATION_IMPLICIT_AUTHENTICATION
+            )
+        },
+        settings = { SettingSection(uiState, onEvent) },
+        result = uiState.result?.let { { ImplicitAuthenticationResult(it) } },
+        action = {
+            AuthenticationButton(
+                uiState.isAuthenticateButtonEnabled,
+                onEvent
+            )
+        }
+    )
+}
+
+@Composable
+fun LoadData(onEvent: (ImplicitAuthenticationViewModel.UiEvent) -> Unit) {
+    LaunchedEffect(Unit) {
+        onEvent.invoke(ImplicitAuthenticationViewModel.UiEvent.LoadData)
+    }
+}
+
+
+
+@Composable
+private fun SettingSection(uiState: ImplicitAuthenticationViewModel.State, onEvent: (ImplicitAuthenticationViewModel.UiEvent) -> Unit) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(Dimensions.verticalSpacing)
+    ) {
+        SdkInitializationSection(uiState.isSdkInitialized)
+        UserProfilesSection(uiState.userProfiles, uiState.selectedUserProfile, onEvent)
+        ScopesSection(uiState.isSdkInitialized, onEvent)
+        AuthenticatedProfileSection(uiState.authenticatedUserProfile)
+
+    }
+}
+
+@Composable
+private fun AuthenticatedProfileSection(userProfile: UserProfile?) {
+    ShowcaseStatusCard(
```

Comment:
Please create new strings, that will explicitly mention implicitly authenticated user. So 

title should be `Implicitly authenticated user profile` and description for empty `No implicitly authenticated user profile`